collect_app/build.gradle

```diff
@@ -57,8 +57,8 @@ def getVersionName = { ->
 }
 
 android {
-    compileSdkVersion(22)
-    buildToolsVersion("25.0.0")
+    compileSdkVersion(25)
```

Comment:
I think it's actually ok and generally a good practice to keep `compileSdkVersion` matching the latest SDK available. It only affects compile time checks.